types/rdfjs__to-ntriples/OTHER_FILES.txt

```diff
@@ -0,0 +1,5 @@
+lib/blankNode.d.ts
+lib/defaultGraph.d.ts
+lib/literal.d.ts
+lib/namedNode.d.ts
+lib/variable.d.ts
```

Comment:
Those files are imported in the tests. I think it should be unnecessary to list them here but the build failed otherwise...

Reply:
weird! I think there might be a bug in the dependency-following code. If I had to guess, it's because the code doesn't handle deep references into scoped packages correctly. I opened https://github.com/microsoft/types-publisher/issues/737 to track it.